modules/openapi-generator/src/main/resources/rust-axum/types.mustache

```diff
@@ -632,6 +632,123 @@ where
     }
 }
 
+
+impl<T> validator::Validate for Nullable<T>
```

Comment:
Basically LGTM. 

However:
- Could you please merge latest master with your branch?
- After that, could you please also re-generate samples & re-run integration test